docs/controls/access_control/ac-05.md

```diff
@@ -0,0 +1,51 @@
+* System owner manages the account access. Additionally, System owner provisions a new user account on request.
+
+Data creation, control, and security functions are completed through the TDP application by a user who has been authenticated and is active in the system. These roles include OFA admin and Data Preppers. 
+* Users from the STTs who will be uploading data will have the role Data Preppers.  Data Preppers collect TANF data and transmit final data to OFA through the TANF application.
+* OFA admins have the ability to upload data on behalf of Data Preppers and upload data files locally into the web application. 
+* These users cannot provision new accounts and do not have access to the account management functionality.
+
+**Part b.**  
+
+**Developers**  
+  - Use GitHub for project planning as well as the hosting site for all code repositories.  
+  - CircleCI ensures all testing, linting and vulnerability scan checks are passed before committing code to the repository or deploy to any environment.  
+  - Ensures a consistent development environment by using Docker to set up a code based container 
+  - Develops code in local environment and tests code in the Dev environment.
+  - Currently, there are two dev environments (*development* and *vendor staging*) in place. The *development* environment is used for internal vendor team review. The *vendor staging* environment is used for government review.  A staging environment (which could be called *Gov Staging*) and a *production* environment will be set up in future releases.The process of dev, stage (testing) and release to production is planned for future releases.  However, all code and documentation committed to HHS has to be approved by the Product Manager and the HHS Tech Lead.  
+
+**Users (OFA Admin and Data Preppers)**
+  - OFA Admin approves new users, updates profile information, inactivates, and reactivates users (Data Preppers (STTs) do not have access to this)
+  - Reads data over assigned region
+  - Able to upload and submit new TANF reports, replace and resubmit TANF reports  
+
+**Part c.**   
+Granting access to the TDP system is performed by the OFA Admin through a request in the application.  Users request access (and create a profile as part of the request) to be able to submit/view reports in the application.
+
+Access to the TDP system occurs in two different ways: through the Django admin interface and through the TDP application interface.  
+
+***Access to the Django admin interface***  
+
+Access directly to the Django interface requires the user have the *superuser* and *staff* flags set and assigned permissions based on responsibilities.  
```

Comment:
How does this relate to the `System owner` role? Which users will have the superuser and staff flags?

Reply:
Updated based on feedback received from Carl.